test/tests/mesh/checkpoint/tests

```diff
@@ -15,6 +15,7 @@
     cli_args = 'Outputs/file_base=test_2a Mesh/file=checkpoint_split_in.2.cpa'
     max_parallel = 2  # workaround until libMesh/libmesh#1408
     min_parallel = 2  # workaround until libMesh/libmesh#1409
+    skip = 'Requires libMesh/libmesh#1409'
```

Comment:
Technically it's #1408 that's required; #1409 was a backwards-compatible change set.

Reply:
OK... Since libMesh/libmesh#1409 includes libMesh/libmesh#1408 and these lines should hopefully be deleted again soon anyway... I'm not going to bother changing it.